docs/config_setup.md

```diff
@@ -5,33 +5,37 @@
 RF24Ethernet requires the RF24 and RF24Network libraries (optionally RF24Mesh) <br>
 See http://tmrh20.github.io for documentation and downloads
 
-See [this video walk-through](https://www.youtube.com/watch?v=rBAIqAaRu0g) of the software setup with Raspberry Pi and Arduino.
+See [this video walk-through](https://www.youtube.com/watch?v=eYiWUTjNSuc) of the software setup with Raspberry Pi and Arduino.
 
 ### Raspberry Pi
 On the Raspberry Pi, a companion program, <a href="http://nRF24.github.io/RF24Gateway/">RF24Gateway</a> must be installed along with the RF24 and RF24Network libraries
 1. Enter the following to download an install script that will build and install the needed RF24* libraries:
    ```shell
-   wget http://tmrh20.github.io/RF24Installer/RPi/install.sh
+   wget https://raw.githubusercontent.com/nRF24/.github/main/installer/install.sh
    chmod +x install.sh
    ./install.sh
    ```
 2. Next, build and run the [RF24Gateway_ncurses example](https://nrf24.github.io/RF24Gateway/RF24Gateway_ncurses_8cpp-example.html)
    ```shell
-   cd RF24Gateway/examples/ncurses
+   cd rf24libs/RF24Gateway/examples/build
    make
-   sudo ./RF24Gateway_ncurses
+   cd ncurses
+   ./RF24Gateway_ncurses
```

Comment:
no more sudo?

Reply:
nope, not needed with the defaults. Don't want to encourage people to run as root if they don't have to.